CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [master]
 ### Added
 
+## [0.0.8] - 2016-01-18
+### Added
+- Support setMailboxes #20
```

Comment:
Please be more precise:

* Added Client.createMailbox
* Added Client.destroyMailbox
etc..